js/tests/test_examples.test.js

```diff
@@ -44,11 +44,10 @@ describe("E2E Tests for plugin demos and examples", () => {
         env: { ...process.env },
         cwd: example.cwd || process.cwd(),
       };
-
       const { stdout, stderr } = await execFileAsync(
         "node",
         [example.file],
-        options
+        { ...options, maxBuffer: 10 * 1024 * 1024 }
```

Comment:
Consider defining the maxBuffer size as a named constant with a comment explaining why 10MB is needed. This helps with maintenance and makes the purpose clear.